README.md

```diff
@@ -282,6 +282,7 @@ have a suggestion for another dataset to include in PyKEEN, please let us know
 | [True Positive Rate](https://en.wikipedia.org/wiki/Sensitivity_(test))                                                 | [0, 1]     | 📈          | TP / (TP + FN)                                              | Classification |
 | [Adjusted Arithmetic Mean Rank (AAMR)](https://arxiv.org/abs/2002.06914)                                               | [0, 2)     | 📉          | The mean over all ranks divided by its expected value.      | Ranking        |
 | [Adjusted Arithmetic Mean Rank Index (AAMRI)](https://arxiv.org/abs/2002.06914)                                        | [-1, 1]    | 📈          | The re-indexed adjusted mean rank (AAMR)                    | Ranking        |
+| [Adjusted Hits at K](https://github.com/pykeen/pykeen/pull/814)                                                        | (inf, 1]   | 📈          | The re-indexed adjusted hits at K                           | Ranking        |
```

Comment:
Is this re-indexed? Or just adjusted? Do we use the two terms consistently?

Reply:
we should rename to adjusted hits at k index

Reply:
the only issue with that is that it would be hard to parse

Reply:
also the lower limit looks wrong - should be smaller than the upper, right?😅

Reply:
okay, it looks like generally the `-` from `-inf` is missing for all rendered value ranges